### PR TITLE
Add styles for hierarchical drawer items

### PR DESCRIPTION
**Pull request by PaskalevStoyan**
closes: https://github.com/telerik/kendo-themes/issues/2946
related to: https://github.com/telerik/k2/issues/471

~~TBD: When the items get too close to the right border of the Drawer, they move on two levels. Q: should they stay as they are now, or the text should overflow?~~ - discussed and delivered 

image

### CODE REVIEW

File: packages/html/src/drawer/tests/drawer-hierarchical-item-levels.tsx
Comment (joneff) [inline packages/html/src/drawer/tests/drawer-hierarchical-item-levels.tsx:39]: Icons must not be clickable.

Add a wrapper around it with the class `k-drawer-toggle`

**Review by joneff — CHANGES_REQUESTED** [review]

File: packages/html/src/drawer/tests/drawer-hierarchical-item-levels.tsx
Comment (PaskalevStoyan) [reply packages/html/src/drawer/tests/drawer-hierarchical-item-levels.tsx]: We have discussed this internally with the Angular team, and currently there is no specific need to add the span wrapper at this point, as all events are attached on the list item and not on the icon itself.

As there is a little time until the code freeze, let's merge this PR as it is and if there is a need to wrap the Icon in a wrapper, we will do so in the future. 
@Raisolution , if i'm missing something, please feel free to fill the gaps.

cc: @Juveniel

**Review by joneff — APPROVED** [review]

**Pull request merged**
